src/components/buildinfo.ts

```diff
@@ -421,14 +424,17 @@ export class BuildInfo {
             const currentAsString = current.toString()
             const endpointAsString = this.currentBuild.pageTotal ? '/' + this.currentBuild.pageTotal.toString(): ''
             const barAsString = this.currentBuild.pageTotal
-                ? generateProgressBar(current / this.currentBuild.pageTotal, this.configuration.get(
+                ? this.generateProgressBar(current / this.currentBuild.pageTotal, this.configuration.get(
                       'progress.barLength'
                   ) as number)
                 : ''
-            this.status.text = `${runIcon}  Page ${padRight(
-                currentAsString + endpointAsString,
-                this.currentBuild.pageTotal ? this.currentBuild.pageTotal.toString().length * 2 + 2 : 6
-            )} ${barAsString}`
+            this.progress.report({
+                message: `${runIcon}  Page ${padRight(
+                        currentAsString + endpointAsString,
+                        this.currentBuild.pageTotal ? this.currentBuild.pageTotal.toString().length * 2 + 2 : 6
+                    )} ${barAsString}`,
+                increment: this.currentBuild.pageTotal ? current / this.currentBuild.pageTotal * 100 : undefined
```

Comment:
`this.currentBuild.pageTotal ?` gives `false` when it is `0`, however `0` is a valid value. I'd suggest using `!== undefined` or similar.

Reply:
That is to avoid division by zero. I do not think we can do anything better.

Reply:
Hmm.
 - if `current === 0` you want `0`
 - if `this.currentBuild.pageTotal > 0` you want `1 / this.currentBuild.pageTotal`
 - otherwise undefined

This seems to be satisfied by
```js
this.currentBuild.pageTotal ? 1 / this.currentBuild.pageTotal :
  current === 0 ? 0 : undefined
```